log4j-perf/src/main/java/org/apache/logging/log4j/message/ParameterFormatterBenchmark.java

```diff
@@ -51,135 +46,83 @@ public static class ThreadState {
         char[] copy = new char[4096];
     }
 
```

Comment:
Why are all these methods deleted?

Reply:
Because they are package-private and not used anymore.